modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java

```diff
@@ -0,0 +1,998 @@
+    // Tenant names and indexes.
+    private static final String PRIMARY_TENANT_NAME = "primary.com";
+    private static final String FEDERATED_TENANT_NAME = "federated.com";
+    private static final int PRIMARY_TENANT = 0;
+    private static final int FEDERATED_TENANT = 1;
+    // Name of the primary Is configured in federated Is as a service provider.
+    private static final String FEDERATED_IS_PRIMARY_SP_NAME = "PrimaryIS";
+    // Primary idp related urls.
+    private static final String PRIMARY_IS_NAME = "PrimaryIS";
+    private static final String PRIMARY_IS_AUTHENTICATOR_NAME_OIDC = "OpenIDConnectAuthenticator";
+    private static final String PRIMARY_IS_AUTHORIZE_ENDPOINT = "https://localhost:9853/t/primary" +
+            ".com/oauth2/authorize";
+    private static final String PRIMARY_IS_TOKEN_ENDPOINT = "https://localhost:9853/t/primary.com/oauth2/token";
+    private static final String PRIMARY_IS_CALLBACK_URL = "https://localhost:9853/t/primary.com/commonauth";
+    //TODO: Enable endpoint in framework.
+    private static final String PRIMARY_IS_BACK_CHANNEL_LOGOUT_ENDPOINT = "https://localhost:9853/t/primary" +
+            ".com/identity/oidc/slo";
+    private static final String PRIMARY_IS_JWKS_URI = "https://localhost:9853/t/primary.com/oauth2/jwks";
+    private static final String PRIMARY_IS_SESSIONS_EXTENSION_ENDPOINT =
+            "https://localhost:9853/t/primary.com/identity/extend-session";
+    // Federated idp related urls.
+    private static final String FEDERATED_IS_AUTHORIZE_ENDPOINT = "https://localhost:9853/t/federated" +
+            ".com/oauth2/authorize";
+    private static final String FEDERATED_IS_CALLBACK_URL = "https://localhost:9853/t/federated.com/commonauth";
+    private static final String FEDERATED_IS_TOKEN_ENDPOINT = "https://localhost:9853/t/federated.com/oauth2/token";
+    private static final String FEDERATED_IS_LOGOUT_ENDPOINT = "https://localhost:9853/t/federated.com/oidc/logout";
+    private static final String FEDERATED_IS_SERVICES_URI = "https://localhost:9853/t/federated.com/services/";
+    private static final String FEDERATED_ME_SESSIONS_ENDPOINT =
+            "https://localhost:9853/t/federated.com/api/users/v1/me/sessions";
+
+    // Urls related to service provider configured in Primary IS.
+    private static final String PRIMARY_IS_SP_NAME = "playground2";
+    private static final String PRIMARY_IS_SP_AUTHENTICATION_TYPE = "federated";
+    private static final String PRIMARY_IS_SP_CALLBACK_URL =
+            "http://localhost:8490/playground2/oauth2-authorize-user.jsp";
+    private static final String PRIMARY_IS_SP_BACK_CHANNEL_LOGOUT_URL = "http://localhost:8490/playground2/bclogout";
+
+    // Urls related to service provider configured in Federated IS.
+    private static final String FEDERATED_IS_SP_NAME = "travelocity";
+    private static final String FEDERATED_IS_SP_CALLBACK_URL = "http://localhost:8490/travelocity.com/home.jsp";
```

Comment:
why do we use travelocity? need to use playground callbackurl for oauth/oidc flows

Reply:
changed the application names to **Application 1** and **Application 2**, and used `https://localhost/` as callbackUrl and back-channel logout url.